circle.yml

```diff
@@ -48,10 +48,10 @@ jobs:
       #    - v1-dep-
 
       - run:
-             name: Installing python dependencies
-             command: |
-             pip install --user --upgrade pip setuptools setuptools-scm
-             pip install --no-cache-dir --user -r bindings/py/packaging/requirements.txt  || exit
+           name: Installing python dependencies
+           command: |
+              pip install --user --upgrade pip setuptools setuptools-scm
+              pip install --no-cache-dir --user -r bindings/py/packaging/requirements.txt  || exit
```

Comment:
needed? If setup.py does it

Reply:
The first one updates pip.  Don't think setup.py does that unless it is  in the requirements file.

Reply:
I think the second one is what causes setup.py to be called.

Reply:
> I think the second one is what causes setup.py to be called. 

It doesn't, just installs the dependencies (as setup.py does too) 
//was talking about this line, the upper needs to stay